packages/modules/viewer-react/src/types.ts

```diff
@@ -61,7 +61,7 @@ export interface ViewerViewCreator3dOptions extends ViewCreator3dOptions {
 
 export interface ViewerFrontstage {
   /** frontstage provider to register */
-  provider: FrontstageProvider;
+  provider: FrontstageProvider; //eslint-disable-line deprecation/deprecation
```

Comment:
Depending on your API versioning policy you might want to introduce: `frontstage: Frontstage` eventually as a replacement for deprecated `provider`.

Reply:
yes, need to review this separately to see how we propagate AppUI deprecations to end users
Not sure if we want to provide backwards compat for now or just move to the new API. 
Further discussion needed and will revisit in separate PR as needed